i18n/zh-cn/user-docs/kubeblocks-for-redis/cluster-management/create-and-connect-to-a-redis-cluster.md

```diff
@@ -50,39 +103,189 @@ import TabItem from '@theme/TabItem';
+   | `spec.componentSpecs`                 | 集群 components 列表，定义了集群 components。该字段允许对集群中的每个 component 进行自定义配置。 |
+   | `spec.componentSpecs.componentDefRef` | 表示 cluster definition 中定义的 component definition 的名称，可通过执行 `kubectl get clusterdefinition redis -o json \| jq '.spec.componentDefs[].name'` 命令获取 component definition 名称。 |
+   | `spec.componentSpecs.name`            | 定义了 component 的名称。  |
+   | `spec.componentSpecs.disableExporter` | 定义了是否开启监控功能。 |
+   | `spec.componentSpecs.replicas`        | 定义了 component 中 replicas 的数量。 |
+   | `spec.componentSpecs.resources`       | 定义了 component 的资源要求。  |
 
-:::note
+   监控 `Cluster` CRD 并创建集群和全部依赖资源。您可执行以下命令获取集群创建的所有资源信息。
 
-* 在生产环境中，不建议将所有副本部署在同一个节点上，因为这可能会降低集群的可用性。
-* 执行以下命令，查看创建 Redis 集群的选项和默认值。
-  
-  ```bash
-  kbcli cluster create redis -h
-  ```
+   ```bash
+   kubectl get all,secret,rolebinding,serviceaccount -l app.kubernetes.io/instance=mycluster -n demo
+   ```
+
+   执行以下命令，查看已创建 Redis 集群的 YAML 文件。
 
-:::
+   ```bash
+   kubectl get cluster mycluster -n demo -o yaml
+   ```
+
+2. 验证集群是否创建成功。
+
+   ```bash
+   kubectl get cluster mycluster -n demo
+   ```
+
+</TabItem>
+
+<TabItem value="kbcli" label="kbcli">
+
+1. 创建 Redis 集群。
+
+   ```bash
+   kbcli cluster create redis mycluster -n demo
+   ```
+
+   如果您需要自定义集群规格，kbcli 也提供了诸多参数，如支持设置引擎版本、终止策略、CPU、内存规格。您可通过在命令结尾添加 `--help` 或 `-h` 来查看具体说明。比如，
+
+   ```bash
+   kbcli cluster create redis --help
+
+   kbcli cluster create redis -h
+   ```
+
+2. 验证集群是否创建成功。
+
+   ```bash
+   kbcli cluster list -n demo
+   >
+   NAME        NAMESPACE   CLUSTER-DEFINITION   VERSION   TERMINATION-POLICY   STATUS     CREATED-TIME
+   mycluster   demo        redis                          Delete               Running    Sep 29,2024 09:46 UTC+0800
+   ```
+
+</TabItem>
+
+</Tabs>
 
 ## 连接到 Redis 集群
 
+<Tabs>
+
+<TabItem value="kubectl" label="kubectl" default>
+
+使用 `kubectl exec` 命令进入 Pod 并连接到数据库。
+
+KubeBlocks operator 会创建一个名为 `mycluster-conn-credential` 的新的 Secret 来存储 MySQL 集群的连接凭证。该 Secret 包含以下 key：
+
+* `username`：Redis 集群的根用户名。
+* `password`：根用户的密码。
+* `port`：Redis 集群的端口。
+* `host`：Redis 集群的主机。
+* `endpoint`：Redis 集群的终端节点，与 `host:port` 相同。
+
+1. 获取用于 `kubectl exec` 命令的 `username` 和 `password`。
+
+   ```bash
+   kubectl get secrets -n demo mycluster-conn-credential -o jsonpath='{.data.username}' | base64 -d
+   >
+   default
+
+   kubectl get secrets -n demo mycluster-conn-credential -o jsonpath='{.data.password}' | base64 -d
+   >
+   5bv7czc4
+   ```
+
+2. 使用用户名和密码，进入 Pod `mycluster-redis-0` 并连接到数据库。
+
+   ```bash
+   kubectl exec -ti -n demo mycluster-redis-0 -- bash
+
+   root@mycluster-redis-0:/# redis-cli -a 5bv7czc4  --user default
+   ```
+
+</TabItem>
+
+<TabItem value="port-forward" label="port-forward">
+
+还可以使用端口转发在本地计算机上连接数据库。
+
+1. 通过端口转发服务。
+  
+   ```bash
+   kubectl port-forward -n demo svc/mycluster-redis 6379:6379
+   ```
+
+2. 在新的终端窗口中执行以下命令，连接到数据库。
+
+   ```bash
+   root@mycluster-redis-0:/# redis-cli -a 5bv7czc4  --user default
```

Comment:
this command is run in a **local** shell, the prompt `root@mycluster-redis-0:/#` is misleading.